Jenkinsfile

```diff
@@ -23,45 +22,19 @@ pipeline {
             agent { label 'master' }
             steps {
                 script {
-                    REPO = JOB_NAME.substring(0, JOB_NAME.indexOf('-build-pr'))
-                    OTHER_REPO = REPO.equals('brave-browser') ? 'brave-core' : 'brave-browser'
                     PLATFORM = JOB_NAME.substring(JOB_NAME.indexOf('-build-pr') + 10, JOB_NAME.indexOf('/PR-'))
                     PIPELINE_NAME = 'pr-brave-browser-' + CHANGE_BRANCH.replace('/', '-') + '-' + PLATFORM
 
-                    if (params.BUILD_STATUS) {
-                        if (Jenkins.instance.getItemByFullName(JOB_NAME).getLastBuild().getCause(hudson.model.Cause$UpstreamCause) == null) {
-                            echo 'Aborting build as it has been started manually with BUILD_STATUS set'
-                            currentBuild.result = 'ABORTED'
-                            return
-                        }
-                        else {
-                            echo "Setting other PR build status to ${params.BUILD_STATUS}"
-                            currentBuild.result = params.BUILD_STATUS
-                            return
-                        }
-                    }
-
                     withCredentials([usernamePassword(credentialsId: 'brave-builds-github-token-for-pr-builder', usernameVariable: 'PR_BUILDER_USER', passwordVariable: 'PR_BUILDER_TOKEN')]) {
                         GITHUB_API = 'https://api.github.com/repos/brave'
                         GITHUB_AUTH_HEADERS = [[name: 'Authorization', value: 'token ' + PR_BUILDER_TOKEN]]
                         CHANGE_BRANCH_ENCODED = java.net.URLEncoder.encode(CHANGE_BRANCH, 'UTF-8')
-                        def prDetails = readJSON(text: httpRequest(url: GITHUB_API + '/' + REPO + '/pulls?head=brave:' + CHANGE_BRANCH_ENCODED, customHeaders: GITHUB_AUTH_HEADERS, quiet: true).content)[0]
+                        def prDetails = readJSON(text: httpRequest(url: GITHUB_API + '/brave-core/pulls?head=brave:' + CHANGE_BRANCH_ENCODED, customHeaders: GITHUB_AUTH_HEADERS, quiet: true).content)[0]
```

Comment:
It's gonna be so nice to not have to bounce between jobs when tracing an issue! 😻